examples/blog/src/app/blog/[slug]/page.tsx

```diff
@@ -0,0 +1,17 @@
+import { BlogPost } from '@/cms'
+
+export default async function BlogArticle({
+  params,
+}: {
+  params: Promise<{ slug: string }>
+}) {
+  const { slug } = await params
+
+  return (
+    <div className="h-full overflow-y-auto">
+      <div className="container mx-auto mt-8">
+        <BlogPost deepdish={{ key: slug }} />
```

Comment:
This is brittle. The `slug` property of the `blogSchema` can ultimately be anything, but it has to match the key exactly AND be URL-safe. It would be nice to support some sort key aliasing or something so that people can have nice slugs correspond 1-1 with a DeepDish key.